### PR TITLE
Task 52242 admin edit profile and header

### PR DESCRIPTION
**Pull request by TMHuy01-08**
## Related Tickets
- [##52242](https://edu-redmine.sun-asterisk.vn/issues/52242)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
Setup router cho admin
Edit profile cho admin
Header cho admin

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by MinhHieu2804** [comment]
approved

**Comment by TMHuy01-08** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -20,6 +20,10 @@ def destroy
   def handle_log_in user
     log_in user
     flash[:success] = t ".login_success"
-    redirect_back_or root_path
+    if user.role_id == 1
```
Comment (ngocvt-0484) [inline app/controllers/sessions_controller.rb]: check theo kiểu enum đi e
user.user?

File: config/routes.rb [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
     post "/login", to: "sessions#create"
     delete "/logout", to: "sessions#destroy"
+
+    namespace :admin do
+      root to: "static_pages#index"
+      resources :static_pages
+      resources :profile, only: %i(edit update)
+    end
   end
+
```
Comment (ngocvt-0484) [inline config/routes.rb]: thừa dòng

**Comment by TMHuy01-08** [comment]
ready

**Pull request merged**